python/res/__init__.py

```diff
@@ -30,8 +30,21 @@
 """
 import os.path
 import sys
-
 import warnings
+import logging
+
+logFormatter = logging.Formatter(
+    '%(levelname)s: %(asctime)s %(name)s %(message)s',
+    "%m/%d - %H:%M:%S"
+)
+
+rootLogger = logging.getLogger()
+
+consoleHandler = logging.StreamHandler()
+consoleHandler.setFormatter(logFormatter)
+consoleHandler.setLevel(logging.DEBUG)
```

Comment:
Are you sure about this one? The `C`-code does the exact opposite:

https://github.com/equinor/libres/blob/59bc1a14b9a18f841395672d7a3e5d5455a5243d/lib/res_util/log.cpp#L100

Reply:
Yes, should be higher than `debug`, would like to set it to `info` for the purpose of this PR, at least temporarily, before this change it was `warning`. Another option is to change to print statements or writing a file, but don't feel like that's a good solution. Ideally I think we would integrate the `python` and `C` logging using the same stream and same levels.